backend/prisma/userSeed.ts

```diff
@@ -0,0 +1,141 @@
+import prisma from '../utils/prismaClient'
+import { RelationshipStatus } from '@prisma/client'
+import crypto from 'crypto'
+const db = prisma
+
+const realUsers = [
+  {
+    email: 'test1@gmail.com',
+    firstName: 'john',
+    lastName: 'smith'
+  }, {
+    email: 'test2@gmail.com',
+    firstName: 'Xavier',
+    lastName: 'Moss'
+  }, {
+    email: 'test3@gmail.com',
+    firstName: 'Kaylee',
+    lastName: 'Ryan'
+  }, {
+    email: 'test4@gmail.com',
+    firstName: 'Scott',
+    lastName: 'Lee'
+  }, {
+    email: 'test5@gmail.com',
+    firstName: 'Carly',
+    lastName: 'Hendrix'
+  }, {
+    email: 'test6@gmail.com',
+    firstName: 'Ella',
+    lastName: 'Haas'
+  }, {
+    email: 'test7@gmail.com',
+    firstName: 'Shania',
+    lastName: 'Baxter'
+  }, {
+    email: 'test8@gmail.com',
+    firstName: 'Halle',
+    lastName: 'Pratt'
+  }, {
+    email: 'test9@gmail.com',
+    firstName: 'Miya',
+    lastName: 'Reid'
+  }, {
+    email: 'test10@gmail.com',
+    firstName: 'Wesley',
+    lastName: 'Lowery'
+  }, {
+    email: 'test111@gmail.com',
+    firstName: 'Bianca',
+    lastName: 'Lawrence'
+  }
+]
+
+const testUser = {
+  email: 'test@gmail.com',
+  firstName: 'bob',
+  lastName: 'admin'
+}
+
+const main = async (): Promise<void> => {
+  // create a test user,
+  try {
+    const salt = crypto.randomBytes(16).toString('base64');
```

Comment:
i won't really block this pr since this is just a db seed script but consider how you could've just written the original production fn in a way that is exportable in a fn such as `createUserWithHashedPw` and then just called that here instead, rather than copy pasting the logic